dll/win32/shell32/shlfileop.cpp

```diff
@@ -710,7 +710,23 @@ static DWORD SHNotifyMoveFileW(FILE_OPERATION *op, LPCWSTR src, LPCWSTR dest, BO
         SHChangeNotify(isdir ? SHCNE_RMDIR : SHCNE_DELETE, SHCNF_PATHW, src, NULL);
         return ERROR_SUCCESS;
     }
+
+#ifdef __REACTOS__
+    DWORD error = GetLastError();
+
+    if (error == ERROR_DISK_FULL && !(op->req->fFlags & (FOF_NOERRORUI | FOF_SILENT)))
+    {
+        CStringW strTitle, strText;
+        strTitle.LoadStringW(IDS_MOVEERRORTITLE);
+        strText.LoadStringW(IDS_DISKFULL);
+
+        MessageBoxW(op->req->hwnd, strText, strTitle, MB_ICONERROR);
+    }
```

Comment:
Since you are adding the same code routines in two functions, it might be better to introduce a separate function instead.

Reply:
just 6 lines, dunno is it worth?

Reply:
done